superflore/generators/ebuild/gen_packages.py

```diff
@@ -30,6 +30,13 @@
 org = "Open Source Robotics Foundation"
 org_license = "BSD"
 
+"""
+This is a blacklist of things that
```

Comment:
Please turn this into a comment with a TODO in it to make it easier to find in the future. This should be something that goes into a config file somewhere in the future.